src/field_theory/normal.lean

```diff
@@ -141,25 +148,22 @@ begin
   let pbED := adjoin_root.power_basis q_irred.ne_zero,
   haveI : finite_dimensional E D := power_basis.finite_dimensional pbED,
   have finrankED : finite_dimensional.finrank E D = q.nat_degree := power_basis.finrank pbED,
```

Comment:
```suggestion
have finrankED : finite_dimensional.finrank E D = q.nat_degree :=
    by rw [power_basis.finrank pbED, adjoin_root.power_basis_dim],
```
Even if this does not create a timeout I think the current proof is abusing a defeq.

Reply:
ahh, I didn't realise this; definitely worth doing.